keras/layers/preprocessing/normalization.py

```diff
@@ -52,11 +52,12 @@ class Normalization(base_preprocessing_layer.PreprocessingLayer):
           example, if shape is `(None, 5)` and `axis=1`, the layer will track 5
           separate mean and variance values for the last axis. If `axis` is set
           to `None`, the layer will normalize all elements in the input by a
-          scalar mean and variance. Defaults to -1, where the last axis of the
+          scalar mean and variance. When `-1` the last axis of the
```

Comment:
I think we should add a ',' after the "When `-1`".

Reply:
Done